projects/control-service/projects/helm_charts/pipelines-control-service/templates/alertmanager.yaml

```diff
@@ -0,0 +1,34 @@
+{{- /*
+  Copyright 2021-2023 VMware, Inc.
+  SPDX-License-Identifier: Apache-2.0
+ */}}
+
+{{- if .Values.alertmanager.enabled }}
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: monitoring.coreos.com/v1alpha1
+    kind: AlertmanagerConfig
+      labels:
+        role: {{ .Values.alertmanager.items.labels.role }}
```

Comment:
the lable key should also be configurable. Keys are custom and not standard. Unless AlertManager config documentation has some established standard keys. 

See how we've made it for ServiceMonitor which is similar in concept: https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/helm_charts/pipelines-control-service/templates/servicemonitor.yaml#L14


Also do add the standard VDK labels - they are added by `{- include "pipelines-control-service.labels" . | nindent 4 }}

Again same as ServiceMonitor.
`